app/routes/account/renew.py

```diff
@@ -0,0 +1,67 @@
+import pexpect
+
+import ocflib.account.validators as validators
+
+from fastapi import HTTPException
+from pydantic import BaseModel, Field
+
+from routes import router
+
+
+class ExpiredPasswordInput(BaseModel):
+    username: str = Field(
+        min_length=3,
+        max_length=16,
+    )
+    old_password: str = Field(
+        min_length=5,
+        max_length=256,
+    )
+    new_password: str = Field(
+        min_length=12,
+        max_length=256,
+    )
+
+
+class ExpiredPasswordOutput(BaseModel):
+    output: str
+    error: str
+
+
+@router.post("/account/renew", tags=["account"], response_model=ExpiredPasswordOutput)
+def reset_password(data: ExpiredPasswordInput):
+    try:
+        validators.validate_username(data.username)
+        validators.validate_password(
+            data.username, data.old_password, strength_check=False
+        )
+        validators.validate_password(
+            data.username, data.new_password, strength_check=True
+        )
+    except ValueError as ex:
+        raise HTTPException(status_code=400, detail=str(ex))
+    cmd = "kinit --no-forwardable -l0 {}@OCF.BERKELEY.EDU".format(data.username)
+    child = pexpect.spawn(cmd, timeout=10)
+    child.expect("{}@OCF.BERKELEY.EDU's Password:".format(data.username))
+    child.sendline(data.old_password)
+    try:
+        result = child.expect(["incorrect", "unknown", pexpect.EOF, "expired"])
+        if result == 0:
+            raise HTTPException(status_code=403, detail="Authentication failed")
```

Comment:
could you use the constants for status codes like here?
https://github.com/ocf/api/blob/f6368bea0fe3257ba2e153ff38db5e60156a7a1c/app/routes/account/change_password.py#L49
it's imported from FastAPI
https://github.com/ocf/api/blob/f6368bea0fe3257ba2e153ff38db5e60156a7a1c/app/routes/account/change_password.py#L9